arm-apimanagement/2016-10-10/swagger/apimapis.json

```diff
@@ -545,12 +545,164 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/operations/{operationId}/policy": {
+      "get": {
+        "tags": [
+          "ApiOperationsPolicy"
+        ],
+        "operationId": "ApiOperationsPolicy_Get",
+        "description": "Get the policy configuration at the API Operation level.",
+        "produces": [
+          "application/vnd.ms-azure-apim.policy+xml"
+        ],
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiIdParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {            
+            "description": "Api Operation Policy information.",
+            "schema": {
+              "type": "file"
+            },
+            "headers": {              
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }              
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ApiOperationsPolicy"
+        ],
+        "operationId": "ApiOperationsPolicy_CreateOrUpdate",
+        "description": "Creates or updates policy configuration for the API Operation level.",
+        "consumes": [
+          "application/vnd.ms-azure-apim.policy+xml"
+        ],
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiIdParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationIdParameter"
+          },
+          {
+            "name": "policyData",
```

Comment:
Standards are now that a body parameter should be named `parameters` (instead of `policyData` ) 
`(BodyParametersValidation/M2063/SDKViolation): A body parameter must be named 'parameters'.`

let's fix these ones 👍 (all cases!)